### PR TITLE
Fixed a bug which could crash rich text editor instances on Firefox

### LINKED ISSUE #884: Sandboxed iframes do not size images correctly on Firefox
**Describe the bug**
When submitting an image as the content of a sandboxed iframe, the first render is clipped vertically.

**To Reproduce**
1. Use Firefox
2. Go to the format editor
3. Select a template
4. Pass an image into a rich text field
5. Click on preview
6. Observe vertically clipped render of image
7. Toggle edit and preview, observe correctly loaded image

**Screenshots**
First render:
image

Second render:
image

**Desktop**
 - OS: Ubuntu 18.04
 - Browser: Firefox
 - Version 74.0

### PR DESCRIPTION
**Pull request by mjvkeulen**
## Description
Moved the destroy hook of TinyMCE over to call remove instead of the destroy API of TinyMCE. This should fix the user interruptions we are experiencing on firefox.

This is still not a clean solution, as the error still occurs (in the TinyMCE logic), however as we do not reuse TinyMCE editor instances, this should fix the problems the user encounters. 

Fixes #884

### CODE REVIEW

File: src/vue/src/components/assets/TextEditor.vue [hunk 1]
```diff
@@ -195,7 +195,12 @@ export default {
         tinymce.init(this.config)
     },
     beforeDestroy () {
-        if (this.editor) { this.editor.destroy() }
+        try {
+            if (this.editor) { tinymce.remove(this.editor) }
+        } catch (NS_ERROR_UNEXPECTED) {
```
Comment (engelhamer) [inline src/vue/src/components/assets/TextEditor.vue]: Why is this catch still needed?

Comment (mjvkeulen) [reply src/vue/src/components/assets/TextEditor.vue]: Sadly as my note says I have no idea atm

File: src/vue/src/components/template/TemplateEdit.vue [hunk 2]
```diff
@@ -19,7 +19,7 @@
             </b-button>
         </div>
         <hr/>
-        <div v-if="mode === 'edit'">
+        <div v-show="mode === 'edit'">
```
Comment (engelhamer) [inline src/vue/src/components/template/TemplateEdit.vue]: Why this?

Comment (Larspolo) [reply src/vue/src/components/template/TemplateEdit.vue]: reduces time to switch between edit and preview. Makes it way better to "quickly check if it is alright"

Comment (Larspolo) [reply src/vue/src/components/template/TemplateEdit.vue]: Can take multiple seconds on non desktops without this

**Pull request merged**